db/migrate/20200206203510_create_review_comments.rb

```diff
@@ -0,0 +1,12 @@
+class CreateReviewComments < ActiveRecord::Migration[6.0]
+  def change
+    create_table :review_comments do |t|
+      t.integer :github_id
+      t.string :body
+
+      t.timestamps
+    end
+
+    add_column :review_comments, :status, :status, default: 'active'
```

Comment:
Any specific reason to add the column outside of the table creation block?

Reply:
The reason is that the type 'status' is a custom type, so if you want to put `t.state` will return an exception that `state` is not a valid type.

Reply:
Just curious, what are you storing there? Why is a custom type needed?